README.md

```diff
@@ -17,9 +17,13 @@ If you really want to build it yourself, the simplest way to do so is to run the
 
 * Open a dev command prompt pointing at the root of the repo.
 * Open the `cppwinrt.sln` solution.
-* Rebuild the x64 Release configuration of the `cppwinrt` project only. Do not attempt to build anything else just yet.
-* Run `build_projection.cmd` in the dev command prompt.
-* Switch to the x64 Debug configuration in Visual Studio and build all projects as needed.
+* Choose a configuration (x64, x86, Release, Debug) and build projects as needed.
```

Comment:
At this point, the whole "Working on the compiler" section may be unnecessary / self-explanatory.